nmap

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+        
```

Comment:
Maybe add a comment here, that this script is to trick the nmap lib for unit testing?